src/components/SignupForm.js

```diff
@@ -53,6 +64,7 @@ const SignupForm = () => {
             bg: (t) => t.colors.primary,
             boxShadow: `0 3px 3px 0 rgba(0, 0, 0, 0.16), 0 3px 3px 0 rgba(0, 0, 0, 0.23)`,
           }}
+          onSubmit={(e) => handleSubmit(values,e)}
```

Comment:
Formik takes in the `handleSubmit` instead of putting it on the `Form`, so we can keep it on line 42 instead

Reply:
yeah I am pretty unsure of best practice with formik but the problem is I needed values to be able to pass into the register = (handlesubmit) function. keeping it on line 42 values is undefined by moving it down into the form I could grab values. this was the solution i came up with definitely interested in learning best way but as its written this will fail if moved to line 42

Reply:
Right, but Formik handles it all for you. It handles the `handleChange` and knows to pass those `values` to your `handleSubmit`. You just have to tell it what to do with those values in the `handleSubmit` you give it on line 67